src/clients/InsightsQueryClient.ts

```diff
@@ -141,6 +141,23 @@ class InsightsQueryClient {
     return response.data as Insights.InsightsQueryResponse<Insights.QueryGetPoolHistoryResponse>;
   }
 
+  async PerpPoolHistory(
+    query: Insights.QueryGetPerpPoolHistoryRequest
+  ): Promise<Insights.InsightsQueryResponse<Insights.QueryGetPerpPoolHistoryResponse>> {
+    const request = this.apiManager.path(
+      "pool/perp/history",
+      {},
+      {
+        limit: query.limit ?? 10,
```

Comment:
default limit can be higher as we might add more perp pools in the future, can set as 100 first?

Reply:
since we are providing both address and poolID, the response length is always just 1.